include/odp/api/spec/packet.h

```diff
@@ -167,6 +167,16 @@ extern "C" {
  * Layer 4 protocol type is ROHC
  */
 
+/**
+ * @typedef odp_packet_vector_t
+ * ODP packet vector
+ */
+
+/**
+ * @def ODP_PACKET_VECT_INVALID
```

Comment:
Should be ODP_PACKET_VECTOR_INVALID (use "vector" instead of "vect" in type names)

Reply:
Will be fixed in next version.